integration/internal/swarm/service.go

```diff
@@ -19,10 +19,10 @@ import (
 // ServicePoll tweaks the pollSettings for `service`
 func ServicePoll(config *poll.Settings) {
 	// Override the default pollSettings for `service` resource here ...
-	config.Timeout = 30 * time.Second
+	config.Timeout = 1 * time.Minute
```

Comment:
I'm not sure if this is the right approach. This PR increases the default poll timeout for services for *ALL* tests, which only `TestServicePlugin` is failing. This could mask real problems later.

Instead, how about an approach like this, specifically for the failing test? https://github.com/moby/moby/pull/36551/

Reply:
The problem is we have tests that were working fine which are now flakey. The reason changing this can cover up real issues is because it seems like the we have some slowdowns in the code and and extending the timeout could be covering up those slowdowns.

Reply:
OK, since we have some concerns about this (though I don't think the timeout failure will really help us some perf issue), let's remove this from this PR.